Timer (lv_timer) example: error: invalid use of incomplete typedef

### LVGL version

9.3.0

### Platform

Handwritten C with FB on Linux (Raspberry)

### What happened?

If i tried this example code:

https://docs.lvgl.io/master/details/main-components/timer.html#creating-a-timer

I got:

`main.c:94:31: error: invalid use of incomplete typedef ‘lv_timer_t’ {aka ‘struct _lv_timer_t’}
   94 |   uint32_t * user_data = timer->user_data;
      |                               ^~
`

### How to reproduce?

`void my_timer(lv_timer_t * timer)
{
  /* Use the user_data */
  uint32_t * user_data = timer->user_data;
  printf("my_timer called with user data: %d\n", *user_data);

  /* Do something with LVGL */
  if(something_happened) {
    something_happened = false;
    lv_button_create(lv_screen_active());
  }
}

...

static uint32_t user_data = 10;
lv_timer_t * timer = lv_timer_create(my_timer, 500,  &user_data);
`

--- a/docs/details/main-components/timer.rst
+++ b/docs/details/main-components/timer.rst
@@ -49,7 +49,7 @@ For example:
    void my_timer(lv_timer_t * timer)
    {
      /* Use the user_data */
-     uint32_t * user_data = timer->user_data;
+     uint32_t * user_data = lv_timer_get_user_data(timer);
      printf("my_timer called with user data: %d\n", *user_data);
 
      /* Do something with LVGL */

